### DISCUSSION TITLE
import a new set of rooms

### DISCUSSION THREAD

**Discussion by ReinSmaragd** [tahoma2d/tahoma2d#1047] [Q&A]
Hi! In Opentoonz it's possible to import a new set of rooms in preferences, you can go to the interface and there you can import a set of "prefab rooms" . I don t see this option in Tahoma. Does this option exist and if ..where can i find it? i
Thks a lot .

**Comment by artisteacher** [comment]
T2D doesn't have multiple sets of rooms - there's just the default. Rooms can still be added and customized as needed. There is also an import option at the bottom of the Preferences sidebar to import rooms & etc. from old installs of T2D.

**Comment by ReinSmaragd** [comment]
Artisteacher thanks very much!
